controllers/complianceeventsapi/server.go

```diff
@@ -0,0 +1,334 @@
+package complianceeventsapi
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha1"
+	"database/sql"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/doug-martin/goqu/v9"
+	_ "github.com/doug-martin/goqu/v9/dialect/postgres" // blank import the dialect driver
+	_ "github.com/lib/pq"
+)
+
+var (
+	clusterKeyCache      sync.Map
+	parentPolicyKeyCache sync.Map
+	policyKeyCache       sync.Map
+)
+
+type databaseAPIServer struct {
+	Lock      *sync.Mutex
+	server    *http.Server
+	isRunning bool
+}
+
+// Start starts the http server. If it is already running, it has no effect.
+func (s *databaseAPIServer) Start(dbURL string) error {
+	s.Lock.Lock()
+	defer s.Lock.Unlock()
+
+	if s.isRunning {
+		return nil
+	}
+
+	mux := http.NewServeMux()
+
+	s.server = &http.Server{
+		Addr:    ":5480",
+		Handler: mux,
+
+		// need to investigate ideal values for these
+		ReadTimeout:  15 * time.Second,
+		WriteTimeout: 15 * time.Second,
+		IdleTimeout:  15 * time.Second,
+	}
+
+	db, err := sql.Open("postgres", dbURL)
+	if err != nil {
+		return err
+	}
+
+	goquDB := goqu.New("postgres", db)
```

Comment:
It kind of looks like the project is on life support. There are a bunch of open PRs with no responses and then there was one release recently in the last two years with one change.

Reply:
That's a good point. The docs were pretty good, and I didn't vet whether it was still very active. Do you think it's bad enough to prevent using it initially here? IMO, the query builder is something we could potentially switch out in a future PR (and the tests don't use the query builder, which could help the swap)

Reply:
Per our conversation, the plan is to merge this with `goqu`, and explore other options. Ideally we remove it soon (either for standard library go only, or another query builder), but we want to un-block anything else that is waiting for the initial API.